dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/stream/TripleServerStream.java

```diff
@@ -336,12 +336,21 @@ private void processHeader(Http2Headers headers, boolean endStream) {
             }
 
             String[] parts = path.split("/");
-            if (parts.length != 3) {
+            String serviceName;
+            String originalMethodName;
+            if (parts.length == 3) {
+                serviceName = parts[1];
+                originalMethodName = parts[2];
+            } else if (parts.length == 4) {
+                if (LOGGER.isWarnEnabled()) {
+                    LOGGER.warn(String.format("context-path: [%s] will be ignored in the triple protocol, please remove this configuration.", parts[1]));
+                }
+                serviceName = parts[2];
+                originalMethodName = parts[3];
+            } else {
```

Comment:
Do fallback here may looks work from remote side but failed on intermediate proxies.
May it be better to just log and `responseErr`?

Reply:
I don't understand what the intermediate proxies refers to. Do you mean that it is better to modify it on the client side?